File: 1913-maximum-product-difference-between-two-pairs/1913-maximum-product-difference-between-two-pairs.py
class Solution:
    def maxProductDifference(self, nums: List[int]) -> int:
        nums.sort()
        x=nums[len(nums)-1]
        y=nums[len(nums)-2]
        a=nums[0]
        b=nums[1]
        return x*y-a*b
    

File: 1913-maximum-product-difference-between-two-pairs/1913-maximum-product-difference-between-two-pairs.java
class Solution {
    
    public int maxProductDifference(int[] nums) {
        int arr[]=new int[nums.length];
        for(int i=0;i<nums.length;i++){
            arr[i]=nums[i];
        }
        Arrays.sort(arr);
        int x=arr[arr.length-1];
        int y=arr[arr.length-2];
        int fi=x*y;
        int a=arr[0];
        int b=arr[1];
        int la=a*b;
        
        return fi-la;
    }
}